packages/proton-bridge/disable_secretservice.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/pkg/keychain/keychain_linux.go b/pkg/keychain/keychain_linux.go
+old mode 100644
+new mode 100755
+index 9376d9a..465d857
+--- a/pkg/keychain/keychain_linux.go
++++ b/pkg/keychain/keychain_linux.go
+@@ -20,7 +20,7 @@ package keychain
+ import (
+ 	"github.com/docker/docker-credential-helpers/credentials"
+ 	"github.com/docker/docker-credential-helpers/pass"
+-	"github.com/docker/docker-credential-helpers/secretservice"
++//	"github.com/docker/docker-credential-helpers/secretservice"
+ )
+ 
+ func newKeychain() (credentials.Helper, error) {
+@@ -30,15 +30,16 @@ func newKeychain() (credentials.Helper, error) {
+ 	if passErr == nil {
+ 		return passHelper, nil
+ 	}
++	return passHelper, nil
+ 
+-	log.Debug("Creating secretservice")
+-	sserviceHelper := &secretservice.Secretservice{}
+-	_, sserviceErr := sserviceHelper.List()
+-	if sserviceErr == nil {
+-		return sserviceHelper, nil
+-	}
++	//log.Debug("Creating secretservice")
++	//sserviceHelper := &secretservice.Secretservice{}
++	//_, sserviceErr := sserviceHelper.List()
++	//if sserviceErr == nil {
++	//	return sserviceHelper, nil
++	//}
+ 
+-	log.Error("No keychain! Pass: ", passErr, ", secretService: ", sserviceErr)
++	//log.Error("No keychain! Pass: ", passErr, ", secretService: ", sserviceErr)
+ 	return nil, ErrNoKeychainInstalled
```

Comment:
Please, explain why.

Reply:
`libsecret` which provides secret service cannot be built on Termux, so I am dropping it from the source to use `pass` only. Pass is well supported on Termux.

Reply:
Okay